app/assets/stylesheets/forever_style_guide/base/_typography.scss

```diff
@@ -101,6 +101,12 @@ a {
   }
 }
 
+.ellipsis_overflow {  // from mars_ui
```

Comment:
Can you add a tech-debt story to remove this from mars ui later, please?

Reply:
Done!